packages/ui-patterns/Cmdk/CommandMenu.tsx

```diff
@@ -85,7 +85,7 @@ const CommandMenu = () => {
     currentPage === COMMAND_ROUTES.AI ||
     currentPage === COMMAND_ROUTES.GENERATE_SQL
       ? 'min(600px, 50vh)'
```

Comment:
Thanks @Hallidayo! Really appreciate you going along with the changes 😄 IIRC it is this line actually that sets the height on the AI pages (it's already bigger than the usual but I guess we can increase it even further)